.github/workflows/update_file.yaml

```
@@ -0,0 +1,55 @@
+# This workflow is to sync dashboards and alert rules from the Sunbeam repo and create a PR if there is an update.
+name: Update files
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '43 1 * * 1'
+
+
+env:
+  REPO_OWNER: canonical
+  REPO_PATH: openstack-exporter-operator
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Pull dashboard and alert rules from sunbeam
+        run: |
+          chmod +x scripts/pull_dashboard_and_alert_rules.sh
+          ./scripts/pull_dashboard_and_alert_rules.sh
```

```suggestion
          bash scripts/pull_dashboard_and_alert_rules.sh
```
if you can't rely on the script being executable, just pass it to the interpreter